service_impl/resolver_service_ce.go

```diff
@@ -6,12 +6,13 @@
 // will be governed by the Apache License, Version 2.0, included in the file
 // licenses/APL2.txt.
 
+//go:build !enterprise
 // +build !enterprise
```

Comment:
Which one is correct? Should we only keep one?

Reply:
Looks like "+build" will be replaced by "go:build" and Goland is insistent on adding "go:build". I will remove "+build".